src/PaymentGateways/Gateways/PayPalCommerce/payPalCommerceGateway.tsx

```diff
@@ -160,7 +160,15 @@ import createSubscriptionPlan from './resources/js/createSubscriptionPlan';
      * @unreleased
      */
     const cardFieldsOnApproveHandler: PayPalCardFieldsComponentBasics['onApprove'] = async (data) => {
-        payPalOrderId = data.orderID;
+        // @ts-ignore
+        const {orderID, liabilityShift} = data;
+        payPalOrderId = orderID
+
+        if (liabilityShift && !['POSSIBLE', 'YES'].includes(liabilityShift)) {
```

Comment:
@jonwaldstein I was checking the link you shared in the PR description and couldn't see any response with **"YES"**.  Usually, they only return **"Y"** instead - check the attached screenshot. 

Also, I can see in [**this other link**](https://developer.paypal.com/docs/checkout/advanced/customize/3d-secure/response-parameters/#supported-parameters) that they said that **"YES"** is used for the `liability_shift` property, but as you can see in the screenshot, they API responses are using the **"Y"** instead. 

So, I'm a bit confused about the allowed values here and wondering if we should include the **"Y"** on this array as well.

Thoughts?

![image](https://github.com/user-attachments/assets/2e3fc4bf-164d-46ef-a001-94bcaa76f7c7)